builtin/providers/aws/resource_aws_iam_server_certificate.go

```diff
@@ -51,9 +51,33 @@ func resourceAwsIAMServerCertificate() *schema.Resource {
 			},
 
 			"name": &schema.Schema{
+				Type:          schema.TypeString,
+				Optional:      true,
+				Computed:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"name_prefix"},
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
+					value := v.(string)
+					if len(value) > 255 {
+						errors = append(errors, fmt.Errorf(
+							"%q cannot be longer than 255 characters", k))
```

Comment:
good catch, I'll update

The server cert name itself is limited to 128, fat-finger copy-paste by me (copied from you, actually :smile: )